### PR TITLE
added helm chart

### PR DESCRIPTION
**Pull request by andrewmoyaccrete**
This pull request is to add a helm chart to GlobaLeaks for deployment to kubernetes. In the root directory, I have added a helm-chart folder that contains the basic necessities needed to deploy GlobaLeaks, using a public docker image of the GlobaLeaks repository. The pull request does not change any other code behavior, and is solely used to deploy the application to kuberenetes. This was tested by first deploying this helm chart to a local kind cluster and watching the status of the pod. No other changes in code were added besides the helm chart directory.

### CODE REVIEW

File: helm-chart/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 0.1.0
+
+# This is the version number of the application being deployed. This version number should be
+# incremented each time you make changes to the application. Versions are not expected to
+# follow Semantic Versioning. They should reflect the version the application is using.
+# It is recommended to use it with quotes.
+appVersion: "1.16.0"
```
Comment (evilaliv3) [inline helm-chart/Chart.yaml:24]: I see many versions; would you please confirm that those are the up to date stable versions?

**Comment by evilaliv3** [comment]
@rglauco @alessiofranceschini : What do you think about this pull request?

**Comment by andrewmoyaccrete** [comment]
Hey @evilaliv3 and @rglauco , I have updated this to a stateful set, added a pvc as well as made the suggested changes as listed above. I have successfully got this to run on a local kind cluster and would expect everything to run smoothly now. Please let me know if any more changes/updates need to be made here, thank you!

File: helm-chart/values.yaml [hunk 2]
```diff
@@ -8,7 +8,7 @@ image:
   repository: andrewmoy/globaleaks
   pullPolicy: IfNotPresent
   # Overrides the image tag whose default is the chart appVersion.
-  tag: helm
+  tag: "4.13.18"
```
Comment (evilaliv3) [inline helm-chart/values.yaml:11]: i wonder if we could create a tag "stable" to be always mapped on the lates stable tag

@rglauco @gianlucagilardi @andrewmoyaccrete : would this be helpful in your opinion?

Comment (andrewmoyaccrete) [reply helm-chart/values.yaml:11]: I am indifferent on this, will do whatever you guys decide

**Comment by andrewmoyaccrete** [comment]
will submit new updated pull request with latest app version with full testing done on k8s cluster

**Pull request closed without merging**